### PR TITLE
Update sockets

### LINKED ISSUE #234: Please update Starscream dependency up to '~> 3.1.1' how it was before in your `v2.7.0` release.
Hello,

In your previous releases `<= v2.7.0` the file: https://github.com/parse-community/ParseLiveQuery-iOS-OSX/blob/2.7.0/ParseLiveQuery.podspec

had the Starscream dependency like:

```
s.dependency 'Starscream', '~> 3.1.0'
```

but in your release `v2.7.1`, see: https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/226

you guys changed it to:
```
s.dependency 'Starscream', '3.1.0'
```

NO `~>` in dependency line unfortunately.

Can you please revert back your changes to `s.dependency 'Starscream', '~> 3.1.0'` or better to `s.dependency 'Starscream', '~> 3.1.1` ?

Because otherwise, if I use another 3'rd party dependency in my project like [Cobrowse.io](https://github.com/cobrowseio/cobrowse-sdk-ios-binary) (which also requires Starscream) then I get pods conflicts error. In this case can't update ParseLiveQuery pod up to latest `v2.7.2`, because the following conflict error shown if I run `pod update`:

```
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Starscream":
  In snapshot (Podfile.lock):
    Starscream (= 3.1.0)

  In Podfile:
    CobrowseIO/Extension was resolved to 2.8.8, which depends on
      Starscream (~> 3.1.1)

    ParseLiveQuery was resolved to 2.7.2, which depends on
      Starscream (= 3.1.0)
```

### PR DESCRIPTION
**Pull request by cbaker6**
- [x] Update Starscream dependency to latest.
- [x] Update to latest Parse-iOS SDK
- [x] Fix versioning in Cocoapods. Close #234

This should be merged after #235 

Should this be 2.8.0? The starscream dependency had a major API change. The updates here weren't too difficult, but will need to see what happens on actual systems. I tested in one of my apps that uses simple liveQueries and works with no issues

### CODE REVIEW

File: Sources/ParseLiveQuery.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -891,7 +891,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (drdaz) [inline Sources/ParseLiveQuery.xcodeproj/project.pbxproj:1259]: I don't have a problem with it, but what's the motivation to bump to 9.0 here?

I think we need to do it when we update to Xcode 12 anyway. But it's currently mismatched with the SDK.

Comment (cbaker6) [reply Sources/ParseLiveQuery.xcodeproj/project.pbxproj:1259]: Right, any dependency that has updated to Xcode 12 requires iOS 9 at a minimum. In this case, Starscream requires the iOS9 bump.

Comment (drdaz) [reply Sources/ParseLiveQuery.xcodeproj/project.pbxproj:1259]: Aha! I see their project file matches. 👍🏼

I only checked their podspec; it's still at 8.0.

Comment (drdaz) [reply Sources/ParseLiveQuery.xcodeproj/project.pbxproj:894]: Actually it looks like they maybe missed updating the iOS version in the podspec. tvOS and iOS have always followed each other, haven't they?

**Review by drdaz — APPROVED** [review]
LGTM

**Comment by cbaker6** [comment]
@TomWFox and @drdaz do you agree that this one should be 2.8.0 and the previous PR 2.7.3?

Technically, I can combine them both for 2.8.0, but since this PR has a bigger dependency change, 2.7.3 would give users the option of having the new features with the old socket dependency

**Comment by drdaz** [comment]
Splitting the changes into two releases would be exceptionally polite to our consumers actually. I like it.

**Comment by TomWFox** [comment]
Sounds good to me.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,15 @@
...
 [Full Changelog](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/compare/2.7.2...master)
 
+### 2.8.0
+
+[Full Changelog](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/compare/2.7.3...2.8.0)
+
+- Bump Starscream dependency to >= 4.0.4 ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6)
+- Bump Parse SDK to v.1.19.1 ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6)
+- Minimum support due to Xcode 12 and dependencies (iOS 9) ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6)
+
```
Comment (TomWFox) [inline CHANGELOG.md]:
```suggestion
[Full Changelog](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/compare/2.8.0...master)

### 2.8.0

[Full Changelog](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/compare/2.7.3...2.8.0)

- Bump Starscream dependency to >= 4.0.4 ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6).
- Bump Parse SDK to v.1.19.1 ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6).
- Minimum support due to Xcode 12 and dependencies (iOS 9) ([#236](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/236)), thanks to [Corey Baker](https://github.com/cbaker6).
```

**Review by TomWFox — DISMISSED** [review]
The version the podspec needs updating too.

**Pull request merged**